crawl/pomparser.py

```diff
@@ -64,6 +64,38 @@ def indent_xml(xml_content, indent):
             handled_indent = True
     return indented_xml
 
+class ParsedProperty:
+
+    def __init__(self, property_name, property_value):
+        self._property_name = property_name
+        self._property_value = property_value
+
+    def get_property_name(self):
+        return self._property_name
+
+    def get_property_value(self):
+        return self._property_value
+
+def parse_version_properties(pom_content):
+    """
+    Parses the <properties> section in the specified pom_content.
+
+    Returns a mapping of a version property name to its actual value.
```

Comment:
Wrong?  Returns a List?

Reply:
Good catch, I had an implementation of using map but I changed it to a list of ParsedProperty. Will update the comments